lib/foreman_maintain/concerns/base_database.rb

```diff
@@ -22,6 +22,40 @@ def ping(config = configuration)
         psql('SELECT 1 as ping', config)
       end
 
+      def file_path_to_backup_db(config = configuration)
+        dump_file_name = "#{config['database']}_#{Time.now.strftime('%Y-%m-%d_%H-%M-%S')}.dump"
+        "#{db_backup_dir}/#{dump_file_name}"
+      end
+
+      def backup_db_command(backup_file_path, config = configuration)
+        "runuser - postgres -c 'pg_dump -Fc #{config['database']} > #{backup_file_path}'"
```

Comment:
Also, I guess it's not a bad idea to filter it though some compression `bzip2 -9 > #{backup_file_path}.bz2`

Reply:
@iNecas,
`-Fc` option also provide some compression.
```
Description of -Fc:
Output a custom-format archive suitable for input into pg_restore. Together with the directory output format, this is the most flexible output format in that it allows manual selection and reordering of archived items during restore. This format is also compressed by default.
```

Please confirm so that I will go & add `bzip2 -9` to command.